static/data-ingestion.yaml

```
@@ -862,29 +533,28 @@ paths:
             schema:
               type: array
               items:
-                $ref: '#/components/schemas/FeedProduct'
+                $ref: '#/components/schemas/FeedProductDelete'
             examples:
               SimpleProductWithImages:
-                summary: Update Product name
-                description: Update Product name
+                summary: Delete Product
+                description: Delete Product
                 value:
                   [
                     {
                       "sku": "red-pants",
-                      "scopes": {
+                      "scope": {
                         "locale": "English"
-                      },
-                      "name": "Red pants - discount!",
-                      "modifiedAt": "2023-06-29 22:30:19"
+                      }
                     }
                   ]
-  /products/v1/prices/{DATA_SPACE_ID}:
-    post:
+
+  /api/v1/pricebooks/{DATA_SPACE_ID}:
+    put:
       tags:
-        - products
-      summary: Product Price Feed API
-      description: Allows managing product prices in the scope of the Website Code and Customer Group Code
-      operationId: PostV1ProductPrices
+        - price-books
+      summary: Pricebooks API create
+      description: Register pricebooks. Pricebook allows you to define specific data for product prices
```

```suggestion
      description: Register pricebooks. PriceBook allows you to define a unique scope that will be used to manage product price discounts across different customer tiers. PriceBook allows to define currency that will be used for all product prices within specified Price Book
```

@akaplya please check description for Price Books ^^

---

Done

---

Done